src/deluge/hid/buttons.h

```diff
@@ -22,12 +22,20 @@
 
 namespace Buttons {
 
+struct button_s {
```

Comment:
I think this can be a class, with `charToButton` as the constructor and `buttonToChar` and `isButton` as (static) member functions.

Reply:
I do want this to interface with C. If I understand correctly, as long as the class doesn't need a vtable the only difference should be default visibility.